src/clients/node/scripts/build_lib.sh

```diff
@@ -24,11 +43,10 @@ else
     echo "Building for '$target'"
 fi
 
-mkdir -p dist
-
 # Need to do string eval-ing because of shellcheck's strict string
 # interpolation rules.
 cmd="./zig/zig build-lib \
+	--global-cache-dir ./zig/global-cache \
```

Comment:
After I moved `install_zig` from a separate npm script to be part of `build_lib`, I got access denied error on cache. I don't understand what was the cause of that difference, but setting global cache to a local dir here helps. 

It also seems like a generally right thing to do -- we don't want to pollute any global dirs with our stuff, zig should be an impl detail of the node package.